src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -389,6 +395,7 @@ case class Event(timestamp: Int,
     case "object" => `object`
     case "value" => value.getOrElse ("")
     case e if e.startsWith ("extra") => extra.getOrElse (Map.empty).getOrElse (e.stripPrefix ("extra."), "")
+    case Event.literalFieldPattern(unquoted) => unquoted
```

Comment:
Slight preference to keep the convention above re: "extra" and use "literal.<value>"

Reply:
Fair. Will change.